os/arch/arm/src/artik053/src/artik053_boot.c

```diff
@@ -128,9 +128,11 @@ static void board_gpio_initialize(void)
 static void board_i2c_initialize(void)
 {
 #ifdef CONFIG_I2C
+#ifdef CONFIG_S5J_I2C
```

Comment:
I cannot understand what means BOARD and CHIP separation.
So below code does not break this separation rule ?
void s5j_board_initialize(void)
{       
#ifdef CONFIG_S5J_SFLASH
        s5j_sflash_init();
#endif
}